quickstarts/browser-segment-investigation/config.yml

```diff
@@ -0,0 +1,38 @@
+# Name of the quickstart (required)
+name: browser-segment-investigation
+
+# Displayed in the UI (required)
+title: Browser Segment Investigation Quickstart 
+
+# Long-form description of the quickstart (required)
+description: |
+ Review the extent of crushed urls so you know which urls you need to add segments for in the browser segment allow list.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+ Check for crushed urls so you can remedy them. 
+
+# Support level: New Relic | Verified | Community (required)
+level: New Relic 
+
+# Authors of the quickstart (required)
+authors:
+  - Kim Hickey 
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - oma 
+  - crushed 
```

Comment:
Suggesting we use a more recognizable term like truncated.

Reply:
@jpvajda I agree there has to be a better term than 'crushed'.  I used this term because it reflects what's used in the field. 
Where else do we use 'truncated'? 
'Collapsed' might be a better term.  What do you think?

Reply:
I didn't see much on "collapsed" either, https://www.google.com/search?q=collapsed+url but if the is common language people are using it's ok to use in this quickstart.